### ISSUE TITLE
Cannot install on Win 10

### ISSUE THREAD

**Issue by pmenico** [derhuerst/sms-cli#1]
I get this error trying to install you project. have you removed this from npm registry?

```
C:\Users\pmeni_000\Documents\twilio\sms-cli>npm install -g sms-cli
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "sms-cli"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/sms-cli
npm ERR! 404
npm ERR! 404  'sms-cli' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\pmeni_000\Documents\twilio\sms-cli\npm-debug.log
```

**Comment by derhuerst** [comment]
I'm sorry that the docs are somewhat misleading. I haven't published this project to npm yet, you can only install it by `npm install sms-cli@derhuerst/sms-cli` for now. Will publish it.

**Comment by derhuerst** [comment]
Published `sms-cli@0.1.0`.

**Comment by pmenico** [comment]
Thanks for prompted response. Now there is a problem with text-prompt@^0.1.1
I tried 0.1.0 and 0.1 but these give same issue

```
C:\Users\pmeni_000\Documents\twilio\sms-cli>npm install -g sms-cli@0.1.0
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "sms-cli@0.1.0"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: text-prompt@^0.1.1
npm ERR! notarget Valid install targets:
npm ERR! notarget 0.1.0
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'sms-cli'
npm ERR! notarget

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\pmeni_000\Documents\twilio\sms-cli\npm-debug.log
```

**Comment by derhuerst** [comment]
`text-prompt@0.1.0` should work. I haven't published `text-prompt@0.1.1` yet, will fix it.

**Comment by derhuerst** [comment]
Published `text-prompt@0.1.1`, please try again.

**Comment by pmenico** [comment]
Getting closer but getting **Error: Cannot find module 'window-size'**. See below.
Is that the correct window-size as there are more than one?
FYI - your README.md says "**todo**" for how to run this
Also, the package.json file seems to be missing a few dependencies

C:\Users\pmeni_000\Documents\twilio\sms-cli>node .

C:\Users\pmeni_000\Documents\twilio\sms-cli>sms help
module.js:471
    throw err;
    ^

**Error: Cannot find module 'window-size'**
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\pmeni_000\AppData\Roaming\npm\node_modules\sms-cli\ui.js:13:27)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

C:\Users\pmeni_000\Documents\twilio\sms-cli>**npm install --save window-size**
sms-cli@0.1.0 C:\Users\pmeni_000\Documents\twilio\sms-cli
`-- window-size@0.3.0


C:\Users\pmeni_000\Documents\twilio\sms-cli>sms help
module.js:471
    throw err;
    ^

Error: Cannot find module 'window-size'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\pmeni_000\AppData\Roaming\npm\node_modules\sms-cli\ui.js:13:27)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

**Comment by derhuerst** [comment]
I guess there will be another thing broken now. :P Published `sms-cli@0.1.1`, please try once more.

**Comment by pmenico** [comment]
We are getting there. Now F-Fetch and D-Delete work, but **R-Respond and C-Compose do not** and give the following error:
F to fetch | ctrl+D to leave | C to compose
(node:49456) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

The way I run this is (after running install):
c:\twilio\sms-cli>node .
c:\twilio\sms-cli>sms

**Comment by pmenico** [comment]
Figured it out - in ui.js query: you have the following line:
			const cmd = (process.env.EDITOR || 'nano')

In windows I do not have 'nano' so I did the following:
c:\twilio\sms-cli>set EDITOR=nodepad.exe
and then re-ran the program and now it woks for C and for R

**Comment by pmenico** [comment]
Just to recap for Windows:
## Installing
cmd
npm install -g sms-cli@0.1.1

## Usage

#load editor and twilio account information to environment
#env.bat contains the following
#set TWILIO_SID=ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
#set TWILIO_TOKEN=29yyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
#set TWILIO_NUMBER=+1aaafffnnnn
#set EDITOR=nodepad.exe
env.bat

#run index.js
node .

#run sms help (does not say much)
sms -help

#run sms system
sms

In notepad type message (or number), save and exit

**Comment by derhuerst** [comment]
I'm willing to make this work on Windows. But I don't have a Windows machine to test on.

If i call `notepad.exe` from the command line, will the command run until I close the window? Also, will I be able to pass a file to edit?

**Comment by pmenico** [comment]
I already got this working. I put notes on githup. Thanks for all your help

**Comment by derhuerst** [comment]
Can you confirm it's working when installing from `master`?

```shell
# either
npm i -g sms-cli@derhuerst/sms-cli
sms

# or
git clone https://github.com/derhuerst/sms-cli.git
cd sms-cli
npm install
./bin.js
```

**Comment by derhuerst** [comment]
I published `sms-cli@0.1.2` to npm.
